rskj-core/src/main/java/co/rsk/core/NetworkStateExporter.java

```diff
@@ -91,8 +91,8 @@ private ObjectNode createContractNode(ContractDetails contractDetails, ObjectNod
     private ObjectNode createAccountNode(ObjectNode mainNode, RskAddress addr, Repository frozenRepository) {
         ObjectNode accountNode = mainNode.objectNode();
         AccountState accountState = frozenRepository.getAccountState(addr);
-        BigInteger balance = accountState.getBalance();
-        accountNode.put("balance", balance.toString());
+        Coin balance = accountState.getBalance();
+        accountNode.put("balance", balance.asBigInteger().toString());
```

Comment:
As documented, the `Coin.toString` method is meant as a debug representation of the coin. Eventually, it will be something like "4 µSBTC". Here, we need the integer representation, thus we're using the `BigInteger` value.

Reply:
Ok, right now the result is the same as aCoin.toString() so I would leave that implementation so when the toString is updated this one is automatically updated.

Reply:
I would rather avoid that, in case someone is parsing the `balance` field as an integer.

Reply:
:+1: